### PR TITLE
chore(plugins): updates examples for new plugin version system

### PR DESCRIPTION
**Pull request by mwagnurr**
relates to OPT-2199

### CODE REVIEW

**Comment by mwagnurr** [comment]
based on #5

File: decision-import-plugin/filter-decision-inputs/README.md [hunk 1]
```diff
@@ -96,16 +99,19 @@ to Optimize as well:
   </build>
 ```
 
-Note: Optimize isolates the loaded plugin libraries from the ones used in Optimize by default.
+Note 1: Optimize will reject plugins that are built with a different Optimize version.
+(for example Optimize 2.6.0 can only run plugins built with version 2.6.0)
+To validate this version it is necessary to create a uber jar so that Optimize can access the plugin environment used by this plugin.
+
+Note 2: Optimize isolates the loaded plugin libraries from the ones used in Optimize by default.
```
Comment (JoHeinem) [inline decision-import-plugin/filter-decision-inputs/README.md]: I think we shouldn't add those notes in the example repository. Let's assume we change something else in the plugin system then we need to change that everywhere in the example repo. I would rather have a hint that says the details of plugin handling can be found in the documentation. Like this we don't have to maintain the hints at several places. Does that make sense?

**Review by JoHeinem — APPROVED** [review]
Very nice! Thanks that you adjusted that and even added sum stuff that was missing before. Just one small thing regarding the notes that you've made. Pre-approving :-)

File: decision-import-plugin/filter-decision-inputs/README.md (hunk 1 above)
Comment (mwagnurr) [reply decision-import-plugin/filter-decision-inputs/README.md]: yepp, I agree, I'll fix that

**Pull request merged**